alpha/alpha-core/src/main/java/org/apache/servicecomb/saga/alpha/core/EventScanner.java

```diff
@@ -79,58 +79,66 @@ private void pollEvents() {
     scheduler.scheduleWithFixedDelay(
         () -> {
           updateTimeoutStatus();
-          findTimeoutEvents();
+          findAllTimeoutEvents();
           abortTimeoutEvents();
           saveUncompensatedEventsToCommands();
           compensate();
           updateCompensatedCommands();
+          markSagaEndedForNoTxEnd();
```

Comment:
for these transaction: 
saga_start->tx_start->tx_abort->tx_abort 
we need to add a saga_end behind them ,so we can judge them finish.
for dumping finished transactions.